lib/git_op.rb

```diff
@@ -17,19 +17,24 @@ def initialize(git_dir, pr)
     @repo_external = ExternalRepoGit.new(pr)
   end
 
-  def ck_or_clone_git(repo)
+  def ck_or_clone_git(repo, https = False)
```

Comment:
Default value is added here, so the current test at tests/t_git_operation.rb can work as it is.

Not sure if we should test there by calling ck_or_clone with https = False and https = True, instead of using the manual test at tests/manual.sh